Pre-migration CQA task for the REST API title (#4937)
theforeman/foreman-documentation: guides/common/modules/ref_listing-hosts.adoc

= Listing hosts

[role="_abstract"]

> You can retrieve a list of all registered hosts, including pagination details and total count.

Reviewer: pagination and total count is not related to the list of hosts; it more about meta data. Maybe shorten it?

```suggestion
You can retrieve a list of all registered hosts from {Project}.
```

Alternative: split into two sentencen; the second sentence could lead with "The {Project} API response includes ...".
